superduper/backends/base/query.py

```diff
@@ -213,6 +270,19 @@ def _set_db(r, db):
             parts.append((part, part_args, part_kwargs))
         self.parts = parts
 
+    def filter(self, *args, **kwargs):
+        """Return a query that filters the documents.
+
+        :param args: The arguments to filter by.
+        :param kwargs: Additional keyword arguments.
+        """
+        query = self if self.parts else self.select()
```

Comment:
Make sure all the filter flavour should be the `select`

Reply:
This function will conflict with the `ibis` implementation of `.filter`

Reply:
self.select() implies `select * from table` right? 

I

Reply:
We will find all the columns and fill in to the args.